community/faas/docs/templates/network.md

```diff
@@ -0,0 +1,40 @@
+# Networks and subnets
+
+Templated network and subnet deployment
+
+## Prerequisites
+- Install [gcloud](https://cloud.google.com/sdk)
+- Create a [GCP project, setup billing, enable requisite APIs](docs/templates/project.md)
+
+
+## Deployment
+
+### Resources
+
+- [compute.v1.network](https://cloud.google.com/compute/docs/reference/latest/networks)
+- [compute.v1.subnetwork](https://cloud.google.com/compute/docs/reference/latest/subnetworks)
+
+
+### Properties
+
+See `properties` section in the schema files
+
+-  [network](../../templates/network.py.schema)
+-  [subnetwork](../../templates/subnetwork.py.schema)
+
+
+### Deployment
+
+#### Create
```

Comment:
I'd like to see a full usage example (how you write the yaml config as well).

Reply:
Yep. Adding links to example configs, and looking further in the existing examples folder for a good full usage example. The one I've used as model are quite skimpy in details (much less than this file). If you have a specific model you want us to follow, please point us to it and we'll use that as a template for these docs.

Reply:
I think the example itself is fine, would just like explanation of how to actually use it. ie. download the template and example, modify which values, etc.